k8s/test/coordinator_service.yaml

```diff
@@ -10,4 +10,4 @@ spec:
       targetPort: 50051
   selector:
     app: coordinator
-  type: LoadBalancer
\ No newline at end of file
+  type: LoadBalancer
```

Comment:
Should this still be an LB or should it be a ClusterIP?

Reply:
this is in /test/ which is just for local dev etc